owncloudApp/src/main/java/com/owncloud/android/shares/ui/ShareActivity.kt

```diff
@@ -144,6 +178,119 @@ class ShareActivity : FileActivity(), ShareFragmentListener {
         getSharesForFileAsyncTask = null
     }
 
+    /**
+     * Updates the view, reading data from [com.owncloud.android.shares.viewmodel.OCShareViewModel]
+     */
+    private fun refreshSharesFromStorageManager() {
+        val shareFileFragment = shareFileFragment
+        if (shareFileFragment?.isAdded == true) {   // only if added to the view hierarchy!!
+            shareFileFragment.refreshUsersOrGroupsListFromDB()
```

Comment:
it would be better to have a livedata listening data from db. You will save to do this approach ;-)

Reply:
This is for private shares and has not been touched yet, it will be our next challenge in the architecture.

Reply:
ok